### DISCUSSION TITLE
[Feature] Add option to share without EXIF metadata

### DISCUSSION THREAD

**Discussion by aaronjamt** [immich-app/immich#9779] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When sharing images, it would be nice to have an option to strip EXIF data for anyone else viewing a file. For instance, I may take a photo at my house and share it with someone, but they would then be able to download the photo and see where I live. This might be a good option to have as a per-user default, or possibly even a per-server default, so users can choose to have EXIF data stripped by default (and then manually be able to choose to instead _not_ strip it when sharing).

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by mmomjian** [comment]
Does disabling "metadata" when making a share link not fit this purpose?

**Reply by bo0tzz** [reply]
It does

**Reply by aaronjamt** [reply]
No, it doesn't. In the UI, sure, the data isn't shown, but when I download the image, it includes all the original EXIF data.

Edit: Just realized that came off as rather standoffish, I apologize. I was saying "No, it doesn't" in response to @mmomjian 's question, not as a way to insult/offend @bo0tzz .

**Reply by mmomjian** [reply]
Got it. Well it looks like we now have this requested in #9835
